### ISSUE TITLE
Version 2.1.13 broken (pry issue)

### ISSUE THREAD

**Issue by jethrocarr** [fairfaxmedia/cloudcycler#3]
The "pry" dependency is missing from the gemspec, this will break the latest version of CloudCycler as per commit 450a21024f678480a3bc30dc5ba874576c3d4ea0

Additionally, once pry is installed, things blow up, looks like we get dropped into some kind of pry debugging mode?

```
/usr/local/bin/cloudcycler -c /etc/cloudcycler.yaml

From: /var/lib/gems/1.9.1/gems/cloudcycler-2.1.13/lib/cloud/cycler/task.rb @ line 289 Cloud::Cycler::Task#cfn_live_stacks:

    284:     end
    285:     # Child stacks are automatically named after their parent. If we do a quick
    286:     # filter for any stack names which have are a prefix of another stack name,
    287:     # we can trim down the number of API requests quite a bit.
    288:     stack_names = stacks.keys.sort
 => 289:     require 'pry'; binding.pry
    290:     0.upto(stack_names.size - 2) do |i|
    291:       stack_name = stack_names[i]
    292:       y = stack_names[i+1]
    293:
    294:       if y.start_with?(stack_name)

[1] pry(#<Cloud::Cycler::Task>)> quit
```

**Comment by jethrocarr** [comment]
Ah so Pry's a debugger... yeah that shouldn't be in there :-)

**Comment by egeland** [comment]
Pry should be a dev-time dependency only.

And yes, it's a debugger, and a pretty cool one at that. :)

**Comment by dbaggerman** [comment]
Oops. Fixed in 2.1.14
